Game2048/Game/Pages/TailContainer.razor

```diff
@@ -0,0 +1,8 @@
+<div class="tile-container">
+    <div class="tile tile-2 tile-position-4-1 tile-new">
+        <div class="tile-inner">2</div>
+    </div>
+    <div class="tile tile-2 tile-position-4-3 tile-new">
+        <div class="tile-inner">2</div>
+    </div>
+</div>
```

Comment:
Also I think this would not be necessary either because we already had the Tile Class with the colors and the Grid already displayed with its elements. Could you explain to us the purpose of this please?